packages/entity-cache-adapter-redis/src/errors/wrapNativeRedisCall.ts

```diff
@@ -0,0 +1,11 @@
+import { EntityCacheAdapterTransientError } from '@expo/entity';
+
+export default async function wrapNativeRedisCall<T>(fn: Promise<T>): Promise<T> {
```

Comment:
```suggestion
export default async function wrapNativeRedisCall<T>(promise: Promise<T>): Promise<T> {
```

This said, a slightly more robust pattern would be to write

```ts
export default async function wrapNativeRedisCall<T>(fn: () => Promise<T>): Promise<T> {

wrapNativeRedisCall(() => this.context.redisClient.mget(...redisKeys));
```

This handles the case where the promise-creating function throws before it first yields. This matters for non-async functions like:
```ts
function fn(): Promise {
  assert(something);
  return new Promise(...);
}

fn(); // throws before we even get back a promise
```